version.txt

```diff
@@ -1 +1 @@
-v8.20.0
+v8.21.0
```

Comment:
it's more a bugfix than an enhancement?

Reply:
Technically it'd be a breaking change

Reply:
yep is a breaking change, so I don't think a patch if correct

Reply:
I know, but this action has been introduced a few days ago and you are the only one using it :D